tests/wdio/validateElement-spec.js

```diff
@@ -0,0 +1,77 @@
+const accessibility = require('../../lib/wdio/services/TerraCommands/accessiblity').default;
+const visualRegressions = require('../../lib/wdio/services/TerraCommands/visual-regression').default;
+
+describe('validateElement', () => {
+  describe('full implementation', () => {
+    before(() => {
+      browser.url('/compare.html');
+      browser.setViewportSize(Terra.viewports('tiny')[0]);
+    });
+
+    Terra.should.validateElement();
+  });
+
+  it('calls the appropriate methods downstream with all arguments', () => {
```

Comment:
I feel like these last two tests would be better suited as jest tests that mock the accessibility and visual regression methods